introduction.md

```diff
@@ -1,3 +1,6 @@
 Carla's Introduction
 My favorite path so far has been HTML/CSS
+* HTML/CSS
+* Video Editing
+* Special Effects and Motion Graphics
 
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?